### ISSUE TITLE
Possible typo/bug v5.07

### ISSUE THREAD

**Issue by XXstack** [kentfredric/HTML-Tree#2]
Have just installed latest HTML::Tree 5.07 (6 days old?) and get this:

Use of uninitialized value $tag in string eq at ....
..../HTML/Element.pm line 1109

Have been using prior versions in same calling code for many many years
without this problem.

**Comment by kentfredric** [comment]
If you can provide a stand-alone piece of code that I can run independently to demonstrate this problem, that would likely accelerate my response to this. :)

**Comment by XXstack** [comment]
Hmm. If it was newish code I would likely be able to hone into that. Unfortunately it is part of a 9 year old suite of some 10,000 lines making heavy use throughout, without much clue as to where the glitch occurs. To make matters worse my suspicion is that only some input material causes it to barf, and can only say that the exact same material did not upset prior versions. I will try (but don't hold your breath!) ......... :)

**Comment by kentfredric** [comment]
Yeah, seems the block of code the error comes from hasn't changed in 7 years, so its probably spooky action at a distance.

https://github.com/kentfredric/HTML-Tree/blame/6ba11171c101895347c5584b5c3b20c4ee0161e2/lib/HTML/Element.pm#L1963-L1982

**Comment by gregoa** [comment]
https://bugs.debian.org/901732 has a reproducer for what seems to be the same issue.
